airbyte-integrations/connectors/source-chargebee/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN pip install .
 
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.16
+LABEL io.airbyte.version=0.2.1
```

Comment:
Why are we skipping 0.2.0?

Reply:
It's in the docker hub already, @artem1205 accidentally pushed the 0.2.0.